templates/NftSaleContract/src/NftSale.cs

```diff
@@ -0,0 +1,81 @@
+using AElf.Contracts.MultiToken;
+using AElf.Sdk.CSharp;
+using AElf.Types;
+using Google.Protobuf.WellKnownTypes;
+
+namespace AElf.Contracts.NftSale
+{
+    // Contract class must inherit the base class generated from the proto file
+    public class NftSale : NftSaleContainer.NftSaleBase
+    {
+        private const string TokenContractAddress = "ASh2Wt7nSEmYqnGxPPzp4pnVDU4uhj1XW9Se5VeZcX2UDdyjx"; // tDVW token contract address
```

Comment:
You can remove this since you are using the token contract system name. Previously in lottery contract, we added this for showcasing contract referencing.